rules-reviewed/eap6/uncategorized/generic-catchall.windup.xml

```diff
@@ -30,10 +30,10 @@
                         </not>
                     </when>
                     <perform>
-                        <classification title="Doug Lea Concurrency util" severity="mandatory">
+                        <classification title="Doug Lea Concurrency util" severity="potential">
```

Comment:
IMO, this one should just be optional. I can't imagine a reason for it to be mandatory for an EAP migration.